go/pkg/client/client.go

```diff
@@ -82,6 +89,15 @@ const (
 	// DefaultMaxBatchDigests is a suggested approximate limit based on current RBE implementation.
 	// Above that BatchUpdateBlobs calls start to exceed a typical minute timeout.
 	DefaultMaxBatchDigests = 4000
+
+	// DefaultDirMode is mode used to create directories.
+	DefaultDirMode = 0700
```

Comment:
It seems like this will change current behaviour from creating dirs / files with 0777 to 0700. This may / may not be problematic for other clients - but at minimum we need to test it before we can sign off on the change.

Wdyt about keeping the existing defaults and rather overriding via params for your use case? That way you can be unblocked and then once we test our other clients, we can change the defaults as well.

Reply:
sounds, good. I changed default.